<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音乐下载器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="center_grid">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="search_tab">
         <attribute name="title">
          <string>搜索</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0,0,0">
              <item>
               <widget class="QLineEdit" name="query_line_edit">
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>单曲/歌手/专辑</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="search_button">
                <property name="text">
                 <string>搜索</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>搜索站点</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="search_site_combobox"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="cloud_music_play_list_line_edit">
                <property name="placeholderText">
                 <string>网易云音乐歌单ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="jump_button">
                <property name="text">
                 <string>跳转</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="search_grid_layout">
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="display_layout">
              <item>
               <widget class="QLabel" name="song_label">
                <property name="text">
                 <string>空空如也~</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="download_button">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>下载</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>download.svg</normaloff>download.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="page_layout"/>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <layout class="QHBoxLayout" name="play_layout"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="download_tab">
         <attribute name="title">
          <string>下载</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="download_grid_layout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="help_menu"/>
  </widget>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>query_line_edit</tabstop>
  <tabstop>cloud_music_play_list_line_edit</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>search_site_combobox</tabstop>
  <tabstop>jump_button</tabstop>
  <tabstop>download_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>query_line_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>search_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cloud_music_play_list_line_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>jump_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>654</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
